### ISSUE TITLE
[Proposal] Revive yearly Rust gamedev survey

### ISSUE THREAD

**Issue by AngelOnFira** [rust-gamedev/wg#106]
A gamedev survey was run 2 years ago by @erlend-sh, the details are in #21. I think it would be cool to try and run one every year around August. It will be cool to see how the ecosystem has evolved, and how questions are answered differently.

@erlend-sh would it be possible to get a list of the questions that were asked during the last survey?

2019 survey results: https://gamedev.rs/blog/survey-01/

**Comment by erlend-sh** [comment]
Sure thing. I’ll send you the questions plus the replies we collected.

I don’t think I came up with the right questions to be asking though. The first time around the survey was quite interesting just by being the first of its kind. The follow-up just didn’t seem to reveal anything particularly interesting. I spoke about this with @alice-i-cecile who had some ideas for how to improve future surveys.

**Comment by alice-i-cecile** [comment]
Absolutely. The core idea here is that you need a plan of "what would I do differently if I knew with perfect certainty what the answer to this question was". The best way to get there is to identify tough, uncertain questions and then target them directly. Examples that come to mind:

1. Should crates be focusing on beginner or advanced documentation?
2. Which areas must be improved before your company is willing to switch to Rust?
3. Which engine (if any) are you most likely to use?
4. Which platforms do you want to make games for?
5. Which licenses (including proprietary arrangements) are you willing to use in your project?

Then, collect some relevant demographics (how long have you been using Rust, are you a professional game developer, what sort of games do you make...) and see if there are strong differences of opinion between groups.

You can also ask more free-form questions for when you need idea-generation, which are basically just posted verbatim. For example: "How do we reduce the ecosystem churn of game engines?" Those questions are sometimes better posed as forum threads, but when it's a particularly sensitive topic the aggregate, semi-anonymous nature of the survey can work better.
